pkg/operator/controller/names.go

```diff
@@ -49,6 +49,13 @@ const (
 	RemoteWorkerLabel = "node.openshift.io/remote-worker"
 )
 
+func AdminGatesConfigMapName() types.NamespacedName {
+	return types.NamespacedName{
+		Name:      "admin-gates",
```

Comment:
We probably want to call this something like:  `openshift-ingress-admingates`

Reply:
I don't think the name of the admin gate configmap can be customizable. [The EP](https://github.com/openshift/enhancements/blob/master/enhancements/update/upgrades-blocking-on-ack.md) mentions only `admin-gates` configmap.